keras/applications/mobilenet.py

```diff
@@ -0,0 +1,275 @@
+'''MobileNet models for Keras.
+# Reference
+- [MobileNets: Efficient Convolutional Neural Networks for
+   Mobile Vision Applications](https://arxiv.org/pdf/1704.04861.pdf))
+'''
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import warnings
+
+from keras.models import Model
+from keras.layers.core import Activation, Dropout, Reshape
+from keras.activations import relu
+from keras.layers.convolutional import Convolution2D, DepthwiseConvolution2D
+from keras.layers.pooling import GlobalAveragePooling2D, GlobalMaxPooling2D
+from keras.layers import Input
+from keras.layers.normalization import BatchNormalization
+from keras.utils.data_utils import get_file
+from keras.engine.topology import get_source_inputs
+from keras.applications.imagenet_utils import _obtain_input_shape
+import keras.backend as K
+
+
+BASE_WEIGHT_PATH = 'https://github.com/titu1994/MobileNetworks/releases/download/v1.0/'
+
+
+def MobileNet(input_shape=None, alpha=1.0, depth_multiplier=1,
+              dropout=1e-3, include_top=True, weights='imagenet',
+              input_tensor=None, pooling=None, classes=1000):
```

Comment:
One keyword argument per line

Why is dropout an argument here? And why so low?